src/main/java/Storage.java

```diff
@@ -0,0 +1,130 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+/**
+ * Represents the storage class, which is responsible for
+ * loading and saving tasks into the save file.
+ */
+public class Storage {
+    private Path saveFilePath;
```

Comment:
Can be final.

Reply:
Thanks for all the suggestions and catching the errors!